party3.tex

```diff
@@ -0,0 +1,35 @@
+\section{Incorporating third party code } \label{sec:party3}
```

Comment:
I've struggled a bit with this section, because I'm not sure how it fits into this document.

Broadly, I think the material on third party code on which we depend (starlink_ast, for example) simply duplicates and restates established policy without adding anything beyond the idea that the DM-CCB should approve future dependencies. I think having it listed here is confusing, and I'd prefer to remove it.

The other material, about including code which builds upon our stack (rather than which our stack is built upon) seems more appropriate for LDM-672 than for this document.

Certainly, little of the below seems related to the release process, so I'm not sure why it's here.

Reply:
I think that that topic deserves a specific document.
LDM-294 may be a good placeholder for that also, or at least mention the topic.

Reply:
I agree with you. So are you removing it from this document?

Reply:
I can try to move it to LDM-294, if @womullan agrees.